pubspec.yaml

```diff
@@ -1,24 +1,18 @@
 name: grinder
-# This version must be updated in tandem with the lib/src/cli.dart version.
```

Comment:
Ugh, I forgot that the dart editor pubspec editor mangles the formatting.  Hopefully we can [fix that](https://github.com/seaneagan/den/issues/17) soon.